draft-ietf-taps-interface.md

```diff
@@ -1776,14 +1776,15 @@ Name:
 : recvChecksumLen
 
 Type:
-: Integer, with special value `Full Coverage`
+: Integer (non-negative with special value `Full Coverage`)
 
 Default:
 : Full Coverage
 
 This property specifies the minimum number of bytes in a received
 message that need to be covered by a checksum. A special value of 0 means
-that no checksum is permitted. A receiving Endpoint will not forward messages to the application
+that a received packet does not need to have a non-zero checksum field. 
+A receiving Endpoint will not forward messages to the application
 that have less coverage. The application is responsible for handling
```

Comment:
Shouldn't this be "A receiving Endpoint will not forward messages that have less coverage to the application." ?